scipy/stats/_continuous_distns.py

```diff
@@ -6942,6 +6942,116 @@ def _stats(self, a):
     def _entropy(self, a):
         return 1 - 1.0/a - np.log(a)
 
+    def fit(self, data, *args, **kwds):
+        data, fshape, floc, fscale = _check_fit_input_parameters(self, data,
+                                                                 args, kwds)
+        # ensure that any fixed parameters don't violate constraints of the
+        # distribution before continuing. The support of the distribution
+        # is `0 < (x - loc)/scale < 1`.
+        if floc is not None:
+            if not data.min() >= floc:
```

Comment:
Do we want strict inequality here? Doesn't equality cause a blow-up? 
Let's  add a test for this one.